qemu/tests/qemu_io_blkdebug.py

```diff
@@ -92,8 +92,8 @@ def run(test, params, env):
         else:
             session.close()
             try:
-                image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
-                output = ""
+                cmd_result = image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
+                output = cmd_result.stdout + cmd_result.stderr
```

Comment:
@PandaWei as said before, the case is for negative test. After trigger the event by delete internal snapshot, qemu-img will exit directly with a error prompt. Need to compare it with the error set in blkdebug.cfg for verify that whether the case passed.